src/common/net.cc

```diff
@@ -248,6 +248,10 @@ Address Address::fromUnix(struct sockaddr *addr) {
   throw Error("Not an IP socket");
 }
 
+Address Address::fromUnix(struct sockaddr_in *addr) {
+  return Address::fromUnix((struct sockaddr *)&addr);
```

Comment:
Please use std::static_cast<> (or whichever cast is approprioate), but not the legacy C style casting.  cppcheck will flag this, as will "-Weffc++".